code-build.js

```diff
@@ -50,7 +58,37 @@ async function waitForBuildEndTime(sdk, { id, logs }, nextToken) {
       cloudWatchLogs
         .getLogEvents({ logGroupName, logStreamName, startFromHead, nextToken })
         .promise()
-  ]);
+  ]).catch(err => {
+    errorDetected = true;
+    errMessage = err;
+
+    return [];
+  });
+
+  if (errorDetected) {
+    //We caught an error in trying to make the AWS api call, and are now checking to see if it was just a rate limiting error
+    if (
+      errMessage.message &&
+      errMessage.message.search("Rate exceeded") !== -1
+    ) {
+      //We were rate-limited, so add `backUp` seconds to the wait time
```

Comment:
Such a nit :) if you can update the comment to `backOff`

Reply:
You're right!  Totally flaked on that